app/models/microcosm.rb

```diff
@@ -0,0 +1,50 @@
+# == Schema Information
+#
+# Table name: microcosms
+#
+#  id           :bigint(8)        not null, primary key
+#  name         :string           not null
+#  description  :text             not null
+#  organizer_id :bigint           not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  slug         :string
+#  location     :string           not null
+#  latitude     :float          not null
+#  longitude    :float          not null
+#  min_lat      :float          not null
+#  max_lat      :float          not null
+#  min_lon      :float          not null
+#  max_lon      :float          not null
+#
+# At this time a microcosm has one organizer.  The first organizer is
+# the user that created the microcosm.
+
+class Microcosm < ApplicationRecord
+  extend FriendlyId
+  friendly_id :name, :use => :slugged
+  self.ignored_columns = ["key"]
+
+  belongs_to :organizer, :class_name => "User"
```

Comment:
Sir gravitystorm, While I disagree in this case (microcosm.organizer tells me something and microcosm.user_id tells me nothing), it doesn't matter.  It will be OBE (overcome by events, not order of the british empire) when membership arrives.